charts/psmdb-db/templates/cluster.yaml

```diff
@@ -15,17 +15,29 @@ spec:
   {{- if .Values.platform }}
   platform: {{ .Values.platform }}
   {{- end }}
-  {{- if .Values.DNSsuffix }}
-  clusterServiceDNSSuffix: {{ .Values.DNSsuffix }}
+  {{- if .Values.clusterServiceDNSSuffix }}
```

Comment:
I have renamed this, so that it is consistent with new `clusterServiceDNSMode`, but maybe we would want here `DNSsuffix` and `DNSmode` instead.

Reply:
I believe it will be better to have the value as in CR.